<launch>
    <arg name="machine" default="prequal_turn" />
    <node name="captain" pkg="mrobosub_planning" type="captain.py" output="screen" args="$(arg machine)">
        <param 
            textfile="$(find mrobosub_planning)/param/globals.yaml" 
            type="yaml"
            name="globals"
        />
        <param 
            textfile="$(find mrobosub_planning)/param/common.yaml" 
            type="yaml"
            name="common"
        />
        <param 
            textfile="$(find mrobosub_planning)/param/competition_states.yaml" 
            type="yaml"
            name="competition_states"
        />
        <param 
            textfile="$(find mrobosub_planning)/param/prequal_front.yaml" 
            type="yaml"
            name="prequal_front"
        />
        <param 
            textfile="$(find mrobosub_planning)/param/prequal_turn.yaml" 
            type="yaml"
            name="prequal_turn"
        />
    </node>
</launch>
